exploration/default-registry-and-mf1-compatibility.md

```diff
@@ -63,37 +63,137 @@ Options:
 
 (When no default is given, the default depends on the locale or implementation)
 
-### Dates and Times
-
-Function name: `:datetime`
-
-Aliases:
-- `:date` (with `style=...` option corresponding to `:datetime dateStyle=...`)
-- `:time` (with `style=...` option corresponding to `:datetime timeStyle=...`)
+---
 
-Operand: "iso8601"
-
-Options:
-- `dateStyle` (`full` `long` `medium` `short`)
-- `timeStyle` (`full` `long` `medium` `short`)
-- `calendar` (buddhist chinese coptic dangi ethioaa ethiopic gregory hebrew indian islamic islamic-umalqura 
-   islamic-tbla islamic-civil islamic-rgsa iso8601 japanese persian roc)
-- `numberingSystem` (arab arabext bali beng deva fullwide gujr guru hanidec khmr knda laoo latn 
-   limb mlym mong mymr orya tamldec telu thai tibt)
-- `timezone` (tzid)
-- `hourCycle` (`h11` `h12` `h23` `h24`)
-- `weekday` (`long` `short` `narrow`)
-- `era` (`long` `short` `narrow`)
-- `year` (`numeric` `2-digit`)
-- `month` (`numeric` `2-digit` `long` `short` `narrow`)
-- `day` (`numeric` `2-digit`)
-- `hour` (`numeric` `2-digit`)
-- `minute` (`numeric` `2-digit`)
-- `second` (`numeric` `2-digit`)
-- `fractionalSecondDigits` (`1`, `2`, `3`)
-- `timeZoneName` (`long` `short` `shortOffset` `longOffset` `shortGeneric` `longGeneric`)
+### Dates and Times
 
-(When no default is given, the default depends on the locale or implementation)
+This subsection describes the functions and options for date/time formatting.
+
+#### Functions
+
+Dates and times have the following functions:
+
+- `:datetime`
+  - defaults to `dateStyle=short timeStyle=short`
+- `:date`
+  - defaults to `dateStyle=short`
+  - Does not permit any of these options:
+    - `timeStyle`
+    - `hour`
+    - `minute`
+    - `second`
+    - `fractionalSecondDigits`
+    - `hourCycle`
+- `:time`
+  - defaults to `timeStyle=short`
+  - Does not permit any of these options:
+    - `dateStyle`
+    - `weekday`
+    - `era`
+    - `year`
+    - `month`
+    - `day`
+
+#### Operands
+
+The operand of a date/time function is either 
+an implementation-defined date/time type (passed in as an argument)
+or a literal that can be parsed to an implementation-defined date/time type.
+
+Date/time literals are required to be an XMLSchema 
+[dateTime](https://www.w3.org/TR/xmlschema11-2/#dateTime),
+[time](https://www.w3.org/TR/xmlschema11-2/#time),
+or [date](https://www.w3.org/TR/xmlschema11-2/#date).
+The `timezoneOffset` of each of these formats is optional. 
+When the offset is not present, conversion to incremental time types is required
+to treat the offset identically to UTC.
```

Comment:
This is highly problematic, as the default timezone uses a different definition. Those two definitions must match so that e.g. `{14:00 :time}` will be formatted as some representation of 2pm, rather than varying depending on the system time's offset from UTC.

Reply:
I fixed this in a different way